### ISSUE TITLE
DeepSpeech superceded by Coqui STT?

### ISSUE THREAD

**Issue by ristew** [RustAudio/deepspeech-rs#48]
It seems like work on the official DeepSpeech project from Mozilla has ended, with no releases since [last December](https://github.com/mozilla/DeepSpeech/releases).  Luckily, though, it appears the team working on it have forked it and continued developing and releasing at https://github.com/coqui-ai/STT.  That being said, should this project change its upstream to Coqui, or should a fork be developed handling it?

**Comment by JRMeyer** [comment]
Coqui STT is well maintained, and upgrading would allow for better models and updates 💚 

https://github.com/mozilla/DeepSpeech/issues/3693

**Comment by est31** [comment]
Yeah defintely, moving to coqui STT would be good. Mozilla has laid off (all? most of?) their employees working on the old deepspeech implementation, and it seems that coqui is doing well. So it might be worth to migrate. I guess this entire repo needs to be renamed?

**Comment by tazz4843** [comment]
Apologies, I made a wrapper and took the `coqui-stt` name on crates.io without looking through the open issues here for a rewrite. 

I'd be willing to transfer the name if desired, but for now I've gotten a basic wrapper at https://crates.io/crates/coqui-stt and https://github.com/tazz4843/coqui-stt.

**Comment by est31** [comment]
Don't worry. I didn't really maintain this well, and I don't really want to do so further. I think it's time to retire this project in favour of yours. I checked out your project. You should probably add examples so that users know how to use it, and github CI so that you know that it works.

**Comment by JRMeyer** [comment]
@est31 -- if you're thinking to archive this project, you might include a link in the README to @tazz4843's new one: https://github.com/tazz4843/coqui-stt
